### PR TITLE
Breaking tests

### PR DESCRIPTION
**Pull request by flavioheleno**

### CODE REVIEW

File: tests/Filter/FilterTest.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ class FilterTest extends \PHPUnit_Framework_TestCase {
 
 	public function testBool() {
 		$expect = array(
-			true,
```
Comment (guilpejon) [inline tests/Filter/FilterTest.php:45]: Hi

Comment (guilpejon) [inline tests/Filter/FilterTest.php:45]: Hello

File: tests/Filter/FilterTest.php [hunk 2]
```diff
@@ -42,7 +42,7 @@ class FilterTest extends \PHPUnit_Framework_TestCase {
...
+			false,
 			true,
 			true,
```
Comment (guilpejon) [inline tests/Filter/FilterTest.php:47]: Hi there

**Review by guilpejon — CHANGES_REQUESTED** [review]

File: tests/Filter/FilterTest.php (hunk 1 above)
Comment (flavioheleno) [inline tests/Filter/FilterTest.php:45]: blablabla

**Pull request closed without merging**